docs/dictionary/control_st/if.lcdoc

```diff
@@ -27,22 +48,33 @@ end if
 
 Parameters:
 condition (bool):
+Any <expression>. (If the <condition> <evaluate|evaluates> to true, 
+the <statement|statements> execute for that <condition>.
+If the <condition> <evaluate|evaluates> to false, the <statement> or 
+<statement list|statementList> following that <condition> is
+skipped.)
 
+statementList:
+Of one or more <LiveCode> <statement|statements>, and can also include
+<if>, <switch>, <try>, or <repeat> <control structure|control
+structures>. 
 
-elseStatementList:
-
+elseIfStatementList:
+Of one or more <LiveCode> <statement|statements>, and can also include
+<if>, <switch>, <try>, or <repeat> <control structure|control
+structures>. 
 
-statementList:
-The <statementList> or <elseStatementList> consists of one or more LiveCode
-statements, and can also include if, switch, try, or repeat control
-structures. The statement or elseStatement consists of a single LiveCode
-statement. 
+elseStatementList:
+Of one or more <LiveCode> <statement|statements>, and can also include
+<if>, <switch>, <try>, or <repeat> <control structure|control
+structures>. 
 
 Description:
-Use the <if> <control structure> to <execute> a <statementList|statement> 
-(or list of <statementList|statements>) only under certain circumstances. 
+Use the <if> <control structure> to <execute> a <statement>
+(or <list of statements|statementList>) only under certain circumstances. 
 
-The <if> <control structure> always begins with the <word> if. There are four forms of the <if> <control structure>:
+The <if> <control structure> always begins with the <word> if. There
```

Comment:
I don't think this `<word>` should be a reference, as it is just the word 'word' rather than the keyword `word`